website/docs/docs/cloud/develop-in-the-cloud.md

```diff
@@ -46,21 +46,21 @@ Read more about IDE updates in the [dbt Cloud release notes](/docs/dbt-versions/
 | **Dark mode**  | From the status bar in the Cloud IDE, enable dark mode for a great viewing experience in low-light environments. 
 
 
-**Start-up process**
+### Start-up process
 
 There are three start-up states when using or launching the Cloud IDE:
 
 - Creation start &mdash; This is the state where you are starting the IDE for the first time. You can also view this as a *cold start* (see below), and you can expect this state to take longer because the git repository is being cloned.
 - Cold start &mdash; This is the process of starting a new develop session, which will be available for you for three hours. The environment automatically turns off three hours after the last activity with the rpc server. This includes compile, preview, or any dbt invocation, however, it *does not* include editing and saving a file.
 - Hot start &mdash; This is the state of resuming an existing or active develop session within three hours of the last activity.
 
-**Work retention**
+### Work retention
 
 The Cloud IDE needs explicit action to save your changes. There are three ways your work is stored:
 
-- Unsaved, local code &mdash; Any code you write is automatically available from your browser’s storage. You can see your changes but will lose them if you switch branches or browsers (another device or browser).
-- Saved but uncommitted code &mdash; When you save a file, the data gets stored in your local storage (EFS storage). If you switch branches but don’t *commit* your saved changes, you will lose your changes.
-- Committed code &mdash; This is stored in the branch with your git provider and you can check out other (remote) branches.
+- **Unsaved, local code &mdash;** Your code is stored only in the browser's local storage. You can see your changes but will lose them if you switch to a different browser. To access the Change branch option, you must "Commit and sync" or "Revert" changes - changing branches isn't available for unsaved code
```

Comment:
### Unsaved, local code
In this state, the user may or may not have saved changes that they need to commit in order to change branches. 
See first screenshot for reference. The change branch option is accessible to me, despite having unsaved changes. This is because all of my saved changes are also committed. However, if a user tries to change branches or revert changes, we will then show a modal warning them that they will lose the unsaved changes (Picture 2)






**Picture 1**
![image](https://user-images.githubusercontent.com/6146772/232492417-54d6473d-e278-462a-9e65-1af833ebcece.png)

**Picture 2**
![image](https://user-images.githubusercontent.com/6146772/232492967-4bda881b-50f2-4c1c-b440-090a40f28f08.png)

Reply:
thanks so much again @davidharting ! oh ok i see! and this is only for code changes. this behavior is the same for file additions or deletions - i thnk this is what i was testing and didn't experience the same behavior. appreciate you clarifying!